distro/superbuild/cmake/externals.cmake

```diff
@@ -224,9 +242,18 @@ endif()
 
 ###############################################################################
 # PythonQt
+set(PythonQt_REPO)
+set(PythonQt_TAG)
+if(${DD_QT_VERSION} VERSION_GREATER "4")
+  set(PythonQt_REPO https://github.com/patmarion/PythonQt.git)
+  set(PythonQt_TAG fix-compile-error-on-qt5.8)
```

Comment:
It can definitively be re-added. It would be nice to work with Florian (from mevislab) and push the patches upstream. I do not have the bandwidth currently ...